lib/addons/amp/index.ts

```diff
@@ -6,6 +6,8 @@ import { AdotCollectorAddOn } from "../adot";
 import { Construct } from 'constructs';
 import { CfnTag } from "aws-cdk-lib/core";
 import { KubectlProvider, ManifestDeployment } from "../helm-addon/kubectl-provider";
+import { KubeStateMetricsAddOn } from '../kube-state-metrics';
```

Comment:
@elamaran11 can you address the Github actions warnings such as 'KubeStateMetricsAddOn' is defined but never used https://github.com/aws-quickstart/cdk-eks-blueprints/actions/runs/3236965644/jobs/5303449934

See lint logs